zoltar-tests/src/test/java/com/spotify/zoltar/fs/FileSystemExtrasTest.java

```diff
@@ -112,4 +113,13 @@ public void copyDirectory() throws IOException, URISyntaxException {
 
     path.toFile().deleteOnExit();
   }
+
+  @Test
+  public void loadFromJar() throws URISyntaxException, IOException {
+    // The .gitignore file includes *.jar, so we've made our test file's extension ._jar (i.e.
+    // added an underscore) so that it is not ignored.
```

Comment:
Hm, this is a bit strange. Should be okay to add a `.jar` despite the `.gitignore`.

Reply:
Agreed the underscore is weird. I changed it to `.jar` and modified the `.gitignore` to not ignore that specific file, which seems cleaner.